proctor-pipet/proctor-pipet-deploy/.gitignore

```diff
@@ -0,0 +1,6 @@
+/build
```

Comment:
Every .gitignore can probably just be condensed into one in the root. They're basically all the same.